boshrelease/pipeline.yml

```diff
@@ -70,8 +70,11 @@ groups:
       - testflight-pr
       - rc
       - shipit
+  - name: versioning
+    jobs:
       - major
       - minor
+      - patch
```

Comment:
Have you felt you wanted to additionally bump the patch aside from after a `shipit`?

Reply:
I've read your notes. I understand.